pyof/v0x01/common/utils.py

```diff
@@ -36,7 +36,7 @@
 from pyof.v0x01.symmetric.echo_request import EchoRequest
 
 
-__all__ = ['new_message_from_header', 'new_message_from_message_type']
+__all__ = ('new_message_from_header', 'new_message_from_message_type')
```

Comment:
Can you explain this, please ? All python docs, including many PEPs, `__all__` is always a list.